Gems/ROS2/Code/Include/ROS2/VehicleDynamics/VehicleInputControlBus.h

```diff
@@ -24,15 +24,27 @@ namespace ROS2::VehicleDynamics
 
         virtual ~VehicleInputControlRequests() = default;
 
+        //! Set target for the vehicle linear speed. It should be realized over time according to drive model.
+        //! @param speedMps is a linear speed in meters per second in evry direction.
```

Comment:
This is a directional argument, so is this description correct? Also mispelled `evry`